firmware/projects/MyBlink/platforms/cli/bindings.cc

```diff
@@ -0,0 +1,31 @@
+#include <unistd.h>
+
+#include <iostream>
+
+#include "../../bindings.h"
+#include "mcal/cli/periph/pwm.h"
+#include "shared/periph/pwm.h"
+#include "mcal/cli/periph/analog_output.h"
+#include "shared/periph/analog_output.h"
+
+namespace mcal {
+
+cli::periph::PWMOutput test{"Test"};
```

Comment:
the periph variable names will be changed (see comment above) but the string names can both be `"Test"` since it will be printed like `Setting AnalogOuput "Test" voltage to ...` or `Setting PWM "Test" duty cycle to ...`